### PR TITLE
[GHO-148][GHO-147][GHO-146][GHO-145][GHO-144]

### PR DESCRIPTION
**Pull request by rupl**
# GHO-148 / GHO-147 / GHO-146 / GHO-145 / GHO-144

- **GHO-148:** Add a bottom separator for Needs and Requirements
- **GHO-147:** When Needs and Requirements sat next to a Hero caption, it created a 2px thick border.
- **GHO-146:** margin/padding adjustments on several components
- **GHO-145:** add padding to Mark author component to make it look nicer next to Hero caption
- **GHO-144:** align titles of Related Articles with absurd precision

### CODE REVIEW

File: html/themes/custom/common_design_subtheme/components/gho-aside/gho-aside.css [hunk 1]
```diff
@@ -21,7 +21,7 @@
 }
 .gho-aside__title {
   margin-top: 0;
-  margin-bottom: 1.5rem;
+  margin-bottom: 2rem;
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/components/gho-aside/gho-aside.css:24]: See main review comment.

File: html/themes/custom/common_design_subtheme/components/gho-facts-and-figures/gho-facts-and-figures.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/**
+ * Override some stuff from gho-aside specifically for Facts and Figures.
+ */
+.gho-facts-and-figures.gho-aside .gho-aside__title {
+  margin-bottom: 1.25rem;
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/components/gho-facts-and-figures/gho-facts-and-figures.css:5]: See main review comment.

**Review by orakili — APPROVED** [review]
Looks good.

**Facts and Figures and Stories**

I interpret the request:

> Top to title: 32 px
> Picture to text: 32 px (Stories), 20 px (Facts and figures)

as changing the margin between the photo and the text below only, keeping the margin between the title and photo unchanged. I guess we'll have to see which interpretation was correct ;-)

**Needs and requirements**

I added 2 commits:

- one to also show the bottom border on small screens because I think it's more consistent:

Before:

Screen Shot 2020-11-25 at 16 03 14

After:

Screen Shot 2020-11-25 at 16 04 08

- The other commit is to avoid the double borders when a NaR is followed by a bottom figures row like on prototype v3 page 22:

Before:

Screen Shot 2020-11-25 at 15 46 16

After:

Screen Shot 2020-11-25 at 15 51 34

**Comment by rupl** [comment]
No matter which interpretation is correct we all lose ;)

**Comment by rupl** [comment]
ah good refinements, tough to check every last combination of content but we're slowly getting there... until we want to remove "just" one separator in one place 😆

**Pull request merged**